### PR TITLE
FAT-2550

### PR DESCRIPTION
**Pull request by nayimovag**
[C10953](https://foliotest.testrail.io/index.php?/cases/view/10953)

https://jenkins-aws.indexdata.com/job/Testing/job/Cypress%20tests/1078

### CODE REVIEW

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/settings/data-export/new-job-profile.cy.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    FieldMappingProfiles.clickNthCheckbox();
+    FieldMappingProfiles.fillInTransformationsTextfields('456', '1', '2', '$a');
+    FieldMappingProfiles.clickTransformationsSaveAndCloseButton();
+    FieldMappingProfiles.clickNewFieldMappingProfileSaveAndCloseButton();
+    // Need to wait until the job is created
+    cy.wait(2000);
+  });
+
+  after('delete user', () => {
+    Users.deleteViaApi(user.userId);
```
Comment (TetianaParanich) [inline cypress/e2e/settings/data-export/new-job-profile.cy.js:43]: Please add deleting created job profile

File: cypress/e2e/settings/data-export/new-job-profile.cy.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    cy.createTempUser([
+      permissions.dataExportEnableSettings.gui,
+      permissions.dataExportEnableApp.gui,
+    ])
+      .then(userProperties => {
+        user = userProperties;
+        cy.login(user.username, user.password, { path: TopMenu.settingsPath, waiter: SettingsPane.waitLoading });
+      });
+
+    FieldMappingProfiles.goTofieldMappingProfilesTab();
```
Comment (TetianaParanich) [inline cypress/e2e/settings/data-export/new-job-profile.cy.js:31]: It's be better if you will create mapping profile by API

Comment (nayimovag) [reply cypress/e2e/settings/data-export/new-job-profile.cy.js]: done

File: cypress/e2e/settings/data-export/new-job-profile.cy.js (hunk 1 above)
Comment (nayimovag) [reply cypress/e2e/settings/data-export/new-job-profile.cy.js:50]: done

**Review by TetianaParanich — APPROVED** [review]

**Pull request merged**